src/nlp.jl

```diff
@@ -296,23 +296,23 @@ function MOI.initialize!(d::NLPEvaluator, requested_features::Vector{Symbol})
         return
     end
 
-    num_variables = numvar(d.m)
+    nvariables = num_variables(d.m)
```

Comment:
How about `num_variables_`. We shouldn't inconsistently use `n` and `num_` to mean the same thing.